windows/ReactTestApp/MainPage.cpp

```diff
@@ -6,27 +6,64 @@
 #include "MainPage.g.cpp"
 #include "Manifest.h"
 
-using winrt::Windows::Foundation::Collections::IVector;
+using namespace ReactTestApp;
+using namespace winrt::Windows::UI::Xaml::Controls;
 
 namespace winrt::ReactTestApp::implementation
 {
     MainPage::MainPage()
     {
         InitializeComponent();
+
+        auto menuItems = MenuFlyout().Items();
+        std::optional<::ReactTestApp::Manifest> manifest = ::ReactTestApp::GetManifest();
+        if (!manifest.has_value()) {
+            MenuFlyoutItem newMenuItem;
+            newMenuItem.Text(L"Couldn't parse app.json");
+            newMenuItem.IsEnabled(false);
+            menuItems.Append(newMenuItem);
+        } else {
+            auto components = manifest.value().components;
+            for (auto &&c : manifest.value().components) {
```

Comment:
Use `components` here:
```suggestion
            for (auto &&c : components) {
```